tasks/env.js

```diff
@@ -28,11 +28,11 @@ module.exports = function(grunt) {
 
     if (this.files.length) {
       if (options.envdir) {
-        var d = _.zipObject(this.files[0].src.map(function(file) {
+        var d = _.fromPairs(this.files[0].src.map(function(file) {
           if (grunt.file.isFile(file)) {
             return [path.basename(file), parse(file)];
           }
-        }));
+        }).filter(function(d) { return d; }));
```

Comment:
what does this filter do? seems unrelated.

Reply:
Need to filter out the null item. Such like [ null, null ]

Reply:
Got it.